mkdocs.yml

```diff
@@ -5,6 +5,10 @@ site_dir: docs
 docs_dir: docs-src
 theme:
   name: material
+  icon:
+    repo: fontawesome/brands/github
+  features:
+    - navigation.tabs
```

Comment:
What does this do?

Reply:
Unfortunately nothing though at one time i believed it would potentially help 🤦  
... sorry about that will fix up. 👍